contrib/internal/httptrace/httptrace_api_gateway_test.go

```diff
@@ -0,0 +1,196 @@
+package httptrace
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+	"strings"
+	"testing"
+
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/mocktracer"
+	"gopkg.in/DataDog/dd-trace-go.v1/internal/normalizer"
+
+	"github.com/stretchr/testify/assert"
+)
+
+var appListener *httptest.Server
+var inferredHeaders = map[string]string{
+	"x-dd-proxy":                 "aws-apigateway",
+	"x-dd-proxy-request-time-ms": "1729780025473",
+	"x-dd-proxy-path":            "/test",
+	"x-dd-proxy-httpmethod":      "GET",
+	"x-dd-proxy-domain-name":     "example.com",
+	"x-dd-proxy-stage":           "dev",
+}
+
+// mock the aws server
+func loadTest(t *testing.T) {
+	// Set environment variables
+	t.Setenv("DD_SERVICE", "aws-server")
+	t.Setenv("DD_TRACE_INFERRED_PROXY_SERVICES_ENABLED", "true")
+
+	// set up http server
+	mux := http.NewServeMux()
+
+	// set routes
+	mux.HandleFunc("/", func(w http.ResponseWriter, r *http.Request) {
+		if r.URL.Path == "/error" {
+			w.WriteHeader(http.StatusInternalServerError)
+			_ = json.NewEncoder(w).Encode(map[string]string{"message": "ERROR"})
+		} else {
+			w.WriteHeader(http.StatusOK)
+			_ = json.NewEncoder(w).Encode(map[string]string{"message": "OK"})
```

Comment:
If we aren't using these variables, why even have an assignment instead of just running the statement? Just remove the left hand side and equals sign